.vib/prometheus/goss/goss.yaml

```diff
@@ -0,0 +1,60 @@
+command:
+  check-no-capabilities:
+    exec: cat /proc/1/status
+    exit-status: 0
+    stdout:
+    - "CapInh:	0000000000000000"
+    - "CapPrm:	0000000000000000"
+    - "CapEff:	0000000000000000"
+    - "CapBnd:	0000000000000000"
+    - "CapAmb:	0000000000000000"
+  {{- $uid := .Vars.server.podSecurityContext.runAsUser }}
+  {{- $gid := .Vars.server.podSecurityContext.fsGroup }}
+  check-user-info:
+    # The UID and GID should always be either the one specified as vars (always a bigger number that the default)
+    # or the one randomly defined by openshift (larger values). Otherwise, the chart is still using the default value.
+    exec: if [ $(id -u) -lt {{ $uid }} ] || [ $(id -G | awk '{print $2}') -lt {{ $gid }} ]; then exit 1; fi
+    exit-status: 0
+  {{ if .Vars.server.serviceAccount.automountServiceAccountToken }}
+  check-sa:
+    exec: cat /var/run/secrets/kubernetes.io/serviceaccount/token | cut -d '.' -f 2 | xargs -I '{}' echo '{}====' | fold -w 4 | sed '$ d' | tr -d '\n' | base64 -d
+    exit-status: 0
+    stdout:
+    - /serviceaccount.*name.*{{.Env.BITNAMI_APP_NAME }}/
+  {{ end }}
+file:
+  /opt/bitnami/prometheus/conf/{{.Vars.server.existingConfigmapKey}}:
+    exists: true
+    contains:
+    - "job_name: alertmanager"
+    - "{{ (first .Vars.server.extraScrapeConfigs).job_name }}"
+  {{.Vars.persistence.mountPath}}:
+    exists: true
+    filetype: directory
+    mode: "2775"
+    owner: root
+  /proc/1/cmdline:
+    exists: true
+    contains:
+    - "--enable-feature={{ .Vars.server.enableFeatures | first }}"
+http:
+  http://localhost:{{.Vars.server.containerPorts.http}}/-/ready:
+    status: 200
+    body:
+    - "Prometheus Server is Ready."
+  http://localhost:{{.Vars.server.containerPorts.http}}/-/healthy:
+    status: 200
+    body:
+    - "Prometheus Server is Healthy."
```

Comment:
Is it possible to check the status of the other servers like alertmanager? Using the service name it should be possible to acess. I would also try accessing via the prometheus server service endpoint (if it exists)